src/pages/VenuePage/VenuePage.tsx

```diff
@@ -231,7 +232,7 @@ const VenuePage: React.FC = () => {
     }
   }
 
-  if (!user) {
+  if (!user || isRelatedVenuesLoading) {
```

Comment:
Hopefully this is possible:
```suggestion
  if (!user) {
```

Reply:
This line was the only reason why the `useRelatedVenues` hook was used in `VenuePage` and if there isn't some hidden specific reason for it being there, it's probaby best to accept the change request.

Reply:
Removed in 4908ac4abd39131d79fdaedea96781e81ebae0cb